application/android/src/co/theengine/loomplayer/LoomSensors.java

```diff
@@ -192,14 +187,15 @@ public void onSensorChanged(SensorEvent event)
                     ///if the sensor data is unreliable return
                     if(event.accuracy == SensorManager.SENSOR_STATUS_UNRELIABLE)
                     {
-                        return;
+                     //   return;
                     }
 
                     ///get correctly oriented values to work with
                     remappedValues = remapXYZValues(event.values);
-                    x = remappedValues[0];
-                    y = remappedValues[1];
-                    z = remappedValues[2];
+                    //normalize the values
+                    x = - remappedValues[0] / 10;
+                    y = - remappedValues[1] / 10;
+                    z = - remappedValues[2] / 10;
```

Comment:
Hmm, why does it need to be negated here, is the remapping wrong or is the source just flipped? Also, why divide by 10? Is this defined as the standard range in API docs and thus mostly consistent device to device?

We should probably define and document what kind of a coordinate system we want first. Range should probably be from -1 to 1. (Inclusive? Exclusive?)